stac_fastapi/pgstac/core.py

```diff
@@ -360,9 +358,6 @@ async def item_collection(
         # If collection does not exist, NotFoundError wil be raised
         await self.get_collection(collection_id, request=request)
 
-        if datetime:
-            datetime = format_datetime_range(datetime)
```

Comment:
can we delete the `format_datetime_range` function now?

Reply:
we could